.vscode/launch.json

```diff
@@ -19,7 +19,7 @@
       "name": "Launch TS",
       "program": "${workspaceFolder}/dist/src/index.js",
       "outFiles": ["${workspaceFolder}/dist/src/**/*.js"],
-      "runtimeExecutable": "/usr/local/bin/node"
```

Comment:
This shouldn't change right? @shrouti1507

Reply:
yes reverting